src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt

```diff
@@ -58,7 +59,7 @@ class SponsorBlockBytecodePatch : BytecodePatch(
         SeekbarFingerprint,
         AppendTimeFingerprint,
         PlayerOverlaysLayoutInitFingerprint,
-        AutoRepeatParentFingerprint,
+        AutoRepeatParentFingerprint, // used for more robust end of video detection (see comments below)
```

Comment:
Comment is not necessary anymore, if it is present below

```suggestion
        AutoRepeatParentFingerprint,
```